peepholelib/models/model_wrap.py

```diff
@@ -208,6 +242,25 @@ def load_checkpoint(self, **kwargs):
         
         return
     
+    def normalize_model(self, **kwargs):
+        '''
+        Wrap the model with an ImageNormalizer layer at the beginning.
+        Args:
+        - mean (torch.tensor): mean for each channel
+        - std (torch.tensor): std for each channel
+        '''
+
+        dss = kwargs['dataset']
+
```

Comment:
add arguments for passing the `mean` and `std` here, and remove the `dss` argument. So the user can import the values from `transforms.py` and then to this method.